src/librustc_errors/lib.rs

```diff
@@ -527,6 +529,7 @@ impl Level {
             Fatal | PhaseFatal | Error => "error",
             Warning => "warning",
             Note => "note",
+            TopLevel => "note",
```

Comment:
It is. I went with this option to avoid breaking existing tooling. If we wan't to make top level comments a general thing, we can give this a differentiated `&str` identifier. If this will only be used for expected/found messages, then I'd rename the enum variant to something else and the identifier can then be `"expected"` and `"found"`. I think that'd be beneficial for tools.